jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -197,10 +198,14 @@ jobs:
       - run: bundle install --path vendor/bundle
 
       # The DB is not automatically created in 2.0, so we manually set up a test DB
-      - run: sudo -u root createuser -h localhost --superuser ubuntu
-      - run: sudo createdb -h localhost test_db
+      # If your project is Rails, then 'db:create' should create a test DB for you.
       - run: bundle exec rake db:create db:schema:load
 
+      # You can also create a test DB manually if your project requires.
+      # We are installing postgresql package because we need 'createdb' command
+      - run: apt-get update -qq; apt-get install -y postgresql
```

Comment:
Is `update` needed here? I know it's probably useful for demonstration, but I think it's unnecessary for documentation (perhaps the user updates earlier in their config, or they will install this directly on their own image).